config/environments/production.rb

```diff
@@ -2,9 +2,6 @@
 
 Rails.application.configure do
   # Settings specified here will take precedence over those in config/application.rb.
-
-  config.active_job.queue_adapter = :delayed_job
```

Comment:
Is this correct?

Reply:
this line was added at the end of file.

https://github.com/CodiTramuntana/decidim-codit_multitenant-app/pull/138/files#diff-da60b4e96eff2b132991226d308949e23f4ef3aad45ad59edd09cbc32cc6251eL58